Core/GDCore/Project/BehaviorConfigurationContainer.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * GDevelop Core
+ * Copyright 2008-2016 Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#include "GDCore/Project/BehaviorConfigurationContainer.h"
+#include <iostream>
+#include "GDCore/Project/PropertyDescriptor.h"
+
+namespace gd {
+
+BehaviorConfigurationContainer::~BehaviorConfigurationContainer(){};
+
+
+std::map<gd::String, gd::PropertyDescriptor> BehaviorConfigurationContainer::GetProperties() const {
+    return GetProperties(content);
+  };
```

Comment:
Should I move this logic in BehaviorJsImplementation like what is done in ObjectJsImplementation?

Reply:
I'm not sure. Can you explain what is "this logic"?

Reply:
ObjectJsImplementation use something like this where it is done with overloading in C++ for behaviors and methods with the content parameter are exposed.
```JS
var newProperties = self['getProperties'](objectContent);
```
An overloading seems easier to understand, but maybe there is some constraints with JS forbidding it.